### PR TITLE
Remove disabled, renderInfo and unit props from Progress

### PR DESCRIPTION
**Pull request by lilyvc**
This PR removes the disabled, renderInfo and unit props from the progress components.

### CODE REVIEW

File: packages/lab/src/progress/LinearProgress/LinearProgress.tsx [hunk 1]
```diff
@@ -90,26 +37,10 @@ export interface LinearProgressProps extends HTMLAttributes<HTMLDivElement> {
...
-      renderInfo,
-      showInfo = true,
-      unit = "%",
-      value = 0,
-      ...rest
-    },
+    { className, max = 100, showInfo = true, value = 0, ...rest },
     ref
   ) {
     const targetWindow = useWindow();
```
Comment (joshwooding) [inline packages/lab/src/progress/LinearProgress/LinearProgress.tsx:46]: progressUnit is not used

Comment (joshwooding) [reply packages/lab/src/progress/LinearProgress/LinearProgress.tsx:46]: progressValue should be inlined

File: packages/lab/src/progress/CircularProgress/CircularProgress.tsx [hunk 2]
```diff
@@ -118,23 +92,12 @@ export const CircularProgress = forwardRef<
...
-      renderInfo(getValueProps())
-    ) : (
-      <DefaultInfo
+    progressInfo = (
+      <Info
         className={withBaseName("progressValue")}
-        unit={unit}
+        unit="%"
         value={Math.round(progress)}
         {...rest}
```
Comment (alycrys) [inline packages/lab/src/progress/CircularProgress/CircularProgress.tsx:102]: why would you pass {...rest} on Info and root element?

File: packages/lab/src/progress/CircularProgress/CircularProgress.tsx [hunk 3]
```diff
@@ -118,23 +92,12 @@ export const CircularProgress = forwardRef<
     }
   }, [ariaLabel]);
```
Comment (alycrys) [inline packages/lab/src/progress/CircularProgress/CircularProgress.tsx:94]: rootStyle is not used

File: packages/lab/src/progress/LinearProgress/LinearProgress.tsx [hunk 4]
```diff
@@ -161,11 +76,7 @@ export const LinearProgress = forwardRef<HTMLDivElement, LinearProgressProps>(
```
Comment (alycrys) [inline packages/lab/src/progress/LinearProgress/LinearProgress.tsx:76]: `rootProps["aria-valuemax"] = max;` maybe?

File: packages/lab/src/progress/CircularProgress/CircularProgress.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (alycrys) [reply packages/lab/src/progress/CircularProgress/CircularProgress.tsx:94]: actually, you're not using it. can be deleted

File: packages/lab/src/progress/LinearProgress/LinearProgress.tsx (hunk 4 above)
Comment (alycrys) [reply packages/lab/src/progress/LinearProgress/LinearProgress.tsx:76]: also, would be great if it was consistent with the code in circular

**Comment by alycrys** [comment]
LGTM, just some small comments

File: packages/lab/src/progress/CircularProgress/CircularProgress.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (lilyvc) [reply packages/lab/src/progress/CircularProgress/CircularProgress.tsx:94]: it's used like 'rootProps["aria-valuenow"] = Math.round(value);'

[the remark this answers is not in the document]
Comment (alycrys) [reply packages/lab/src/progress/CircularProgress/CircularProgress.tsx:94]: true. But you're not passing rootProps. Instead you're passing these directly 
```
      aria-valuemax={max}
      aria-valuemin={0}
      aria-valuenow={value}
```

[the remark this answers is not in the document]
Comment (lilyvc) [reply packages/lab/src/progress/CircularProgress/CircularProgress.tsx:94]: is this appropriate const rootProps: HTMLAttributes<'div'> = {}? not sure if div is relevant. seems like i can put any string there

File: packages/lab/src/progress/LinearProgress/LinearProgress.tsx (hunk 4 above)
Comment (lilyvc) [reply packages/lab/src/progress/LinearProgress/LinearProgress.tsx:72]: thanks, updated

File: packages/lab/src/progress/CircularProgress/CircularProgress.tsx (hunk 2 above)
Comment (lilyvc) [reply packages/lab/src/progress/CircularProgress/CircularProgress.tsx]: removed from info

**Comment by lilyvc** [comment]
> LGTM, just some small comments

thanks, hopefully amended now

**Review by alycrys — APPROVED** [review]

**Pull request merged**